packages/rancher-monitoring/rancher-monitoring/generated-changes/overlay/templates/rancher-monitoring/dashboards/rancher/performance-dashboards.yaml

```diff
@@ -0,0 +1,17 @@
+{{- if and .Values.grafana.enabled .Values.grafana.defaultDashboardsEnabled }}
```

Comment:
this should only be added if rancherServiceMetrics is enabled right?

Reply:
@aiyengar2 I don't think it really matters, worse case you can look at the dashboard, it is just empty, but I can add it. Although there is a value I think most people would toggle the feature using the environment variable.

Reply:
People would follow this guide to turn it on: https://github.com/rancher/rancher/wiki/How-to-setup-prometheus-dashboard-for-controllers

We def don't want them turning it on/off by upgrading the chart. However it could be default off, then an upgrade to on for initial capture, then use env var from there on out. Majority of people, I would assume 95% or more, would not use this graph. 

The new check is on `.Values.rancherServiceMetrics.enabled` which we just need to test

Reply:
Added the `.Values.rancherServiceMetrics.enabled` check, tested, and appears to work.

Reply:
@rmweir thanks, sounds good.

@cbron to be clear (since I'm not sure if that was your impression), turning on or off the Grafana dashboard won't cause you to lose any data. The data is persisted in Prometheus; Grafana is just making a call to Prometheus and presenting it in a fancy dashboard. So upgrading the chart and no longer seeing the dashboard followed by upgrading the chart to enable it again would only cause you to lose data for the time that you turned it off and then turned it back on (because the ServiceMonitor disappeared, not because the dashboard did).